test/regress/run_regression.py

```diff
@@ -245,12 +245,16 @@ def run(self, benchmark_info):
 
 class DumpTester(Tester):
     def applies(self, benchmark_info):
-        return benchmark_info.expected_exit_status == EXIT_OK
+        return (
+            benchmark_info.benchmark_ext != ".p"
+            and benchmark_info.expected_exit_status == EXIT_OK
+            and "Parse Error" not in benchmark_info.expected_output
```

Comment:
Checking for `"Parse Error"` in the expected output seems rather fragile. Can we instead explicitly disable all of those tests?

You should be able to do something like:

```
 grep -lr "Parse Error" test/regress/regress[0-4] | xargs sed -i '1 i\; DISABLE-TESTER: dump'
```